### PR TITLE
Feature/insulation boxes

### PR DESCRIPTION
**Pull request by Andreas-Tonnessen**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [x] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/DisciplineReleaseControl/Components/Garden/ReleaseControlGardenItem.tsx [hunk 1]
```diff
@@ -48,6 +48,7 @@ const ReleaseControlGardenItem = ({
                     )}
                     <StatusCircle
                         statusColor={getGardenItemCompletionColor(data.completionStatus)}
+                        size={12}
```
Comment (Noggling) [inline src/apps/DisciplineReleaseControl/Components/Garden/ReleaseControlGardenItem.tsx:51]: Har vi en standared her?

File: src/apps/DisciplineReleaseControl/Components/Garden/StatusCircle.tsx [hunk 2]
```diff
@@ -4,17 +4,18 @@ import { tokens } from '@equinor/eds-tokens';
...
+    height: ${(props) => (props.size ? props.size + 'px' : '14px')};
     outline: 1px solid ${tokens.colors.text.static_icons__primary_white.rgba};
     background-color: ${(props) => props.statusColor};
     border-radius: 50%;
     margin-left: 4px;
 `;
 
-export const StatusCircle = ({ statusColor }: StatusIconProps): JSX.Element => (
-    <Circle statusColor={statusColor} />
+export const StatusCircle = ({ statusColor, size }: StatusIconProps): JSX.Element => (
```
Comment (Noggling) [inline src/apps/DisciplineReleaseControl/Components/Garden/StatusCircle.tsx:19]: Felles komponent?

File: src/apps/DisciplineReleaseControl/Components/Sidesheet/CheckListTable.tsx [hunk 3]
```diff
@@ -39,7 +39,10 @@ export const CheckListTable = ({ checkLists }: TableProps): JSX.Element => {
     return (
         <>
             <h4>Pipetest checklists:</h4>
-            <Table options={{ columns: columns, data: checkLists }}></Table>
+            <Table
+                options={{ columns: columns, data: checkLists }}
+                height={35 + checkLists?.length * 35}
```
Comment (Noggling) [inline src/apps/DisciplineReleaseControl/Components/Sidesheet/CheckListTable.tsx:44]: Kan du navngi 35 her? padding?

File: src/apps/DisciplineReleaseControl/Components/Sidesheet/ReleaseControlSidesheet.tsx [hunk 4]
```diff
@@ -102,4 +136,21 @@ export const ReleaseControlSidesheet = ({
 
 const TablesTab = styled.div`
     padding-left: 8px;
+    overflow: auto;
```
Comment (Noggling) [inline src/apps/DisciplineReleaseControl/Components/Sidesheet/ReleaseControlSidesheet.tsx:139]: Flytte til egen style fil?

File: src/apps/DisciplineReleaseControl/Components/Workflow/Components/WorkflowWarningTriangle.tsx [hunk 5]
```diff
@@ -56,28 +56,28 @@ export const WarningTriangleOutline = styled.div`
...
     width: 0;
     height: 0;
     border-style: solid;
     position: relative;
-    border-width: 0 5px 10px 5px;
+    border-width: 0 4.5px 9px 4.5px;
     border-color: transparent transparent #eb0000 transparent;
-    right: 5px;
-    top: 3px;
+    right: 4.5px;
```
Comment (Noggling) [inline src/apps/DisciplineReleaseControl/Components/Workflow/Components/WorkflowWarningTriangle.tsx:69]: egen file og tokens?

**Review by Noggling — APPROVED** [review]
Litt små plukk ;) men eller ok

File: src/apps/DisciplineReleaseControl/Components/Garden/ReleaseControlGardenItem.tsx (hunk 1 above)
Comment (Andreas-Tonnessen) [reply src/apps/DisciplineReleaseControl/Components/Garden/ReleaseControlGardenItem.tsx:51]: 14 er "standard", men noen steder vil det brukes 12

**Pull request merged**